Makefile

```diff
@@ -128,6 +128,11 @@ kind:
 	kind create cluster --name dc2 --image $(KIND_NODE_IMAGE)
 
 
+# Helper target for loading local dev images (run with `DEV_IMAGE=...` to load non-k8s images)
+kind-load:
+	kind load docker-image --name dc1 $(DEV_IMAGE)
+	kind load docker-image --name dc2 $(DEV_IMAGE)
```

Comment:
For this backport, I left out the `dc3` and `dc4` load commands as the change that introduced those DC's wasn't backported to 0.49.x.